packages/sdk/src/sdk/api/users/UsersApi.ts

```diff
@@ -393,4 +395,70 @@ export class UsersApi extends GeneratedUsersApi {
     })
     return { ethWallet, userBank }
   }
+
+  /** @hidden
+   * Add an encrypted email for a user
+   */
+  async addEmail(params: EmailRequest, advancedOptions?: AdvancedOptions) {
+    const {
+      emailOwnerUserId,
+      primaryUserId,
+      encryptedEmail,
+      encryptedKey,
```

Comment:
where we will be doing the key creation?

Reply:
in this same function in a future PR